<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>Номер запроса</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Название крана</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Максимальная высота</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Максимальная грузоподъемность</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Местоположение</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ответственный</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Аренда</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Сортировать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="Connection"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="Disconnection"/>
    <addaction name="separator"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Редактирование</string>
    </property>
    <addaction name="Add"/>
    <addaction name="Delete"/>
    <addaction name="Edit"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Дополнительно</string>
    </property>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Язык</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Connection"/>
   <addaction name="Disconnection"/>
   <addaction name="separator"/>
   <addaction name="Edit"/>
   <addaction name="Add"/>
   <addaction name="Delete"/>
   <addaction name="separator"/>
   <addaction name="Exit"/>
  </widget>
  <action name="Connection">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/rec/img/lan_check_icon_137457.png</normaloff>:/rec/img/lan_check_icon_137457.png</iconset>
   </property>
   <property name="text">
    <string>Соединение с БД</string>
   </property>
  </action>
  <action name="Disconnection">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/rec/img/lan_disconnect_icon_138437.png</normaloff>:/rec/img/lan_disconnect_icon_138437.png</iconset>
   </property>
   <property name="text">
    <string> Отключиться от БД</string>
   </property>
  </action>
  <action name="Exit">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/rec/img/exit_icon-icons.com_70975.png</normaloff>:/rec/img/exit_icon-icons.com_70975.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="Add">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/rec/img/plus_icon-icons.com_70890.png</normaloff>:/rec/img/plus_icon-icons.com_70890.png</iconset>
   </property>
   <property name="text">
    <string>Добавить строку</string>
   </property>
  </action>
  <action name="Delete">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/rec/img/minus-symbol_icon-icons.com_73485.png</normaloff>:/rec/img/minus-symbol_icon-icons.com_73485.png</iconset>
   </property>
   <property name="text">
    <string>Удалить строку</string>
   </property>
  </action>
  <action name="Edit">
   <property name="icon">
    <iconset resource="resourses.qrc">
     <normaloff>:/rec/img/353430-checkbox-edit-pen-pencil_107516.png</normaloff>:/rec/img/353430-checkbox-edit-pen-pencil_107516.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать данные</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Об Авторе</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Русский</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Английский</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Итальянский</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
